src/main/java/com/example/todolist/TaskController.java

```diff
@@ -0,0 +1,44 @@
+package com.example.todolist;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@CrossOrigin(origins = "http://localhost:3000")
+public class TaskController {
+
+    private final TaskService taskService;
+
+    public TaskController(TaskService taskService) {
+        this.taskService = taskService;
+    }
+
+    @GetMapping("/taskList")
+    public ResponseEntity<List<Task>> taskList() {
+       List<Task> data = taskService.getAllTasks();
+       return ResponseEntity.ok(data);
+    }
+
+    @DeleteMapping("/deleteTask")
+    public ResponseEntity<String> deleteTask(@RequestParam(name = "index") Long id) {
+        try {
+            taskService.deleteTask(id);
+            return ResponseEntity.ok().build();
+        } catch (Exception e) {
```

Comment:
Create Custom Exceptions and controller them with Spring Global Exception handler (https://medium.com/@aedemirsen/spring-boot-global-exception-handler-842d7143cf2a)

Reply:
https://github.com/Ramya300/Task/commit/8ae4b88ed1d5b1de1e0e07902d5d1f58baea49b4

updated with custom exceptions. refer the commit attached.

@govardhankoyaraju

Reply:
@govardhankoyaraju